src/RevisionPopupWidget.js

```diff
@@ -112,81 +81,119 @@ function getUserLinksContent( data ) {
 }
 
 /**
- * Get content for the loading state.
- * @return {jQuery}
+ * Initialize the widget by building the content structure.
  */
-function getLoadingStateContent() {
-	const $shimmerSpan = $( '<span>' )
-		.addClass( 'wwt-shimmer wwt-shimmer-animation' );
-	return $( '<div>' )
-		.append( $shimmerSpan, $shimmerSpan.clone(), $shimmerSpan.clone() );
-}
+RevisionPopupWidget.prototype.initialize = function () {
+	this.diffSizeLabel = new OO.ui.LabelWidget( {
+		classes: [ 'mw-diff-bytes' ]
+	} );
+
+	this.commentLabel = new OO.ui.LabelWidget( {
+		classes: [
+			'comment',
+			'comment--without-parentheses'
```

Comment:
If this is initialised here with this class name, then empty comments still get parentheses. It was being done within a check e.g. `if ( data.comment !== '' ) { … }`. Although I guess the fix here isn't to not create this here, but rather to make sure it's removed when there's no comment?

Reply:
omg **that's** why there are empty parentheses! I could not find those anywhere and it drove me nuts.

Noted! Will add them after the comment is there (and if there is a comment at all)